src/autopredictor/fit.py

```diff
@@ -0,0 +1,25 @@
+def fit(X_train,X_test,y_train,y_test):
+
+    """
+    Train and evaluate multiple regression models on the given training and test data.
```

Comment:
Should we specify the models that we will be fitting?

Reply:
I don't think so because we are not yet sure which ones we are using. I feel like we can change them for Milestone 2.

Reply:
Sounds good!